sql/flow_config.go

```diff
@@ -78,7 +78,7 @@ func GetPypiVersion(configURL, astroCliVersion string) (AstroSQLCliVersion, erro
 			}
 		}
 	}
-	return AstroSQLCliVersion{SQLCliCompatibility.SQLCliVersion, SQLCliCompatibility.PreRelease}, nil
+	return AstroSQLCliVersion{SQLCliCompatibility.SQLCliVersion, true}, nil
```

Comment:
```suggestion
	return AstroSQLCliVersion{SQLCliCompatibility.SQLCliVersion, SQLCliCompatibility.PreRelease}, nil
```
Need to release SQL CLI 0.4.0 and revert this before merging the PR. Made temporary change so that tests could fetch the pre-release.